interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/emails/telehealth-invitation-existing.html.twig

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html
+        PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml"
+      xmlns:o="urn:schemas-microsoft-com:office:office">
+
+<head>
+    <title>{{ title }}</title>
```

Comment:
`xlt`

Reply:
haha, I didn't think of escaping the html in the email invitations.  Make's sense since the email client is likely running some mini webkit engine that is likely to have the same vulnerabilities as the browser.  Thanks for catching me on this.

Reply:
Fixed